lib/module.js

```diff
@@ -48,9 +40,9 @@ function Module(id, parent) {
 }
 module.exports = Module;
 
-Module._cache = {};
-Module._pathCache = {};
-Module._extensions = {};
+Module._cache = Object.create(null);
+Module._pathCache = Object.create(null);
+Module._extensions = Object.create(null);
```

Comment:
Yeah I started citgm, but it looks like there may be some infrastructure issues, since there was some trouble on downloading npm packages on pretty much every citgm machine.